support/install_command.ps1

```diff
@@ -43,11 +43,20 @@ Function Download-Chef($url, $sha256, $dst) {
 
 Function Install-Chef($msi) {
   Log "Installing Chef Omnibus package $msi"
-  $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi" -Passthru -Wait
-  $p.WaitForExit()
-
-  if ($p.ExitCode -ne 0) { throw "msiexec was not successful. Received exit code $($p.ExitCode)" }
-
+  $installingChef = $True
+  $installAttempts = 0
+  while ($installingChef) {
+    $installAttempts++
+    $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi" -Passthru -Wait
+    $p.WaitForExit()
+    if ($p.ExitCode -eq 1618) {
+      Log "Another msi install is in progress (exit code 1618), retrying ($($installAttempts))..."
+      continue
```

Comment:
As it stands, it already takes a few seconds between installs, since it takes the MSI a second to register that another install is in progress.